collect_app/src/main/res/layout/project_settings_dialog_layout.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:background="@drawable/project_settings_dialog_background"
```

Comment:
@lognaturel how do you feel about us not using the super rounded shape from the mockups? I think we should just use the standard dialog shape here. As far as I understand shape can be customized at a theme level (https://material.io/design/shape/about-shape.html) and the extra rounded corners is probably part of Google's overall look and feel.

Reply:
The rounded shape is from https://material.io/components/dialogs#alert-dialog, https://www.figma.com/file/3UJK55PPAvNBdYC7pjbQ6r/%E2%9D%96-Material-Theming-%2B--Android-11-(Community)?node-id=1081%3A8808.

I don't feel strongly about it and agree we should have one style for dialogs as much as possible. If it's straightforward to make a blanket change for all dialogs, I think it's a small thing that will make the app feel more modern.

Reply:
I think we should have a separate issue for styling all dialogs and then remove the custom styling I implemented.

Reply:
@grzesiek2010 that sounds good. Are you able to file?

Reply:
Sure: https://github.com/getodk/collect/issues/4482